api/Request.json

```diff
@@ -586,7 +586,8 @@
             },
             "firefox": [
               {
-                "version_added": "39"
+                "version_added": "39",
+                "notes": "In Firefox 61, the default value of <code>credentials</code> changed from <code>\"omit\"</code> to <code>\"same-origin\"</code>."
```

Comment:
It would be clearer and easier to scan (IMO) to represent this using an explicit version range, like:

```
            "firefox": [
              {
                "version_added": "39"
              },
              {
                "version_added": "34",
                "version_removed": "61",
                "notes": [
                  "Default value is <code>omit</code>."
                ]
              },
              {
                "version_added": "34",
                "flags": [
                  {
                    "type": "preference",
                    "name": "dom.fetch.enabled"
                  }
                ]
              }
            ]
```
That would get a rendered table like this:

<img width="583" alt="screen shot 2018-05-25 at 11 49 12 am" src="https://user-images.githubusercontent.com/432915/40561353-bb2d81f8-6011-11e8-90b2-eade934dc585.png">

Reply:
That's different from the way fscholz recommended I do the same thing yesterday, to note a similar change to a function, where it continued to exist but its value situation changed. See https://github.com/mdn/browser-compat-data/pull/2149/commits/a63df8bde437fc4bacb194c6b32473f6f2cf73dd. Let me know if this really does need changing and I'll do that (although I'm running very low on time before I head out on vacation).

Reply:
I'll go with whatever @Elchi3 wants to do, this is just my opinion.

Reply:
OK, I"m going to let you guys sort it out and I will let you fix it so I can worry about other things leading up to departing for a week. :)

Reply:
A third way would be to add a sub feature, so that there will be new row rendered with the behavior change. I expect that this change will occur in other browser versions, too, so there will be more versions for this to record. You would have something like: 
- "Basic support" (for the initial spec)
- "Default value <code>same-origin</code>" (for the updated spec)

I think this is actually the way I was preferring in the past. Sometimes changes are so minor, that I was fine with a note, though. We've only recently discovered to use version ranges more and I agree they also make things nicer to scan. Here, my preference would be to go with the sub feature so that we have another row in the table, but I'm curious to hear what others think.

Reply:
> I expect that this change will occur in other browser versions, too, so there will be more versions to record for this.

If other browsers were going to change here, then I agree, a subfeature would be best.

But I read this situation as:

* the default in the standard is `same-origin`: this is documented as the default in the main body of the page

* `same-origin` is the default for all browsers except Firefox

* `omit` was the default in Firefox (contra the standard) and this was fixed in 61

If this is correct, then I wouldn't expect this to occur for other browsers. If it's not correct, the docs are misleading (to me anyway).

Reply:
I think it is a change in the standard. See this PR and the particular comment with bugs to all browsers: https://github.com/whatwg/fetch/pull/585#issuecomment-325351254